apps/api/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,6 @@
-export const environment = {
-  production: true
+import { Environment } from './environment.interface';
+
+export const environment: Environment = {
+  production: true,
+  dbUri: process.env.MONGODB_URI
```

Comment:
didn't know we could use process variable here

Reply:
Yeah, it's very handy

Reply:
ahh, this is the API, sorry, I got confused with frontend env vars